eng/ci/templates/jobs/build.yml

```diff
@@ -0,0 +1,53 @@
+parameters:
+- name: pool
+  type: string
+  default: '1es-pool-azfunc'
+
+jobs:
+  - job: Build
+    displayName: 'Build'
+
+    pool:
+      name: ${{ parameters.pool }}
+      image: 1es-ubuntu-22.04
+      os: linux
+
+    variables:
+      NODE_VERSION: '20.x'
+
+    steps:
+    - checkout: self
+
+    - task: UseNode@1
+      inputs:
+        version: ${{ variables.NODE_VERSION }}
+      displayName: 'Setup Node'
+
+    - task: Npm@1
+      inputs:
+        command: 'install'
+      displayName: 'Install dependencies'
+
+    - task: Npm@1
+      inputs:
+        command: 'custom'
+        customCommand: 'run build'
+      displayName: 'Build'
+
+    - bash: |
```

Comment:
Do you not need to [install java](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/java-tool-installer-v0?view=azure-pipelines) before this?